proxy/outprocess/inc/module_loaders/outprocess_module.h

```diff
@@ -33,6 +33,8 @@ typedef struct OUTPROCESS_MODULE_CONFIG_DATA
     STRING_HANDLE message_uri;
     /** @brief Arguments for the module .*/
     STRING_HANDLE outprocess_module_args;
+	/** @brief controls timeout for ipc retries. */
+	int default_wait;
```

Comment:
I put the "timeout" in the loader entrypoint of the JSON where the out of process configuration goes (like the control and message channel ids, but I still need to pass it to the module so it can use the timeout.